lib/scenes/level_scene.cpp

```diff
@@ -93,6 +95,26 @@ void LevelScene::performPrepare() {
             gadget_spawns[i].always_respawn);
         entity_components->push_back(std::move(comps));
     }
+
+    // Create billboards
+    for(Json billboard : json.getVector("billboards")) {
+        Billboard bb_obj = Billboard();
+
+        bb_obj.x = billboard.getInt("x");
+        bb_obj.y = billboard.getInt("y");
+        bb_obj.alpha = billboard.getInt("alpha");
+
+        bb_obj.x_scale = billboard.getInt("x_scale");
+        bb_obj.y_scale = billboard.getInt("y_scale");
+
+        bb_obj.content_path = billboard.getString("content_path");
+        struct stat buffer;   
+        if (stat (bb_obj.content_path.c_str(), &buffer) != 0)
+            // The advertisement image does not exist
+            bb_obj.content_path = "advertisement/pisswasser.png";
```

Comment:
Yo m8, this is unix only, use this:
https://en.cppreference.com/w/cpp/filesystem/exists